### PR TITLE
Add `getCurrentCategoryNumber` function

### PR DESCRIPTION
**Pull request by trizin**

### CODE REVIEW

File: backend/routes/project.js [hunk 1]
```diff
@@ -110,6 +111,12 @@ router.post(
           return res.status(400).send("Proposal already exists for this round");
         }
 
+        const categoryId =
+          process.env.DEVELOPMENT_CATEGORY_ID ?? (await getCurrentCategoryId());
```
Comment (idiom-bytes) [inline backend/routes/project.js:115]: <3

**Comment by idiom-bytes** [comment]
Maybe getCurrentCategoryId() should be named getCurrentDiscourseCategoryId()?

File: backend/utils/airtable/utils.js [hunk 2]
```diff
@@ -70,6 +70,14 @@ async function getCurrentRoundNumber() {
   return roundParameters ? roundParameters[0].fields["Round"] : -1;
 }
 
+async function getCurrentCategoryId() {
```
Comment (idiom-bytes) [inline backend/utils/airtable/utils.js:73]: Maybe getCurrentCategoryId() should be named getCurrentDiscourseCategoryId()? Category is also used in proposals.

Comment (trizin) [reply backend/utils/airtable/utils.js]: Done!

**Pull request merged**